ping/rust/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rand = "0.8"
 serde = {version = "1", features = ["derive"]}
 serde_json = "1"
 soketto = "0.7.1"
-testground = {git = "https://github.com/testground/sdk-rust", branch = "master", version = "0.4.0"}
+testground = {git = "https://github.com/testground/sdk-rust", rev = "94a9a72796f94cc7ca786a5f019d07f328c76d4b", version = "0.4.0"}
```

Comment:
Just saw this change while working on #42. Was this a temporary change because until #42 is merged we are not compatible with the master branch, or should I also use a concrete revision there?

Reply:
I just released `testground` `v0.4.0`. How about using that instead?

Reply:
https://github.com/testground/sdk-rust/pull/38